### PR TITLE
fix the docs of docker ps help message

### PR DESCRIPTION
**Pull request by jimmyxian**
Fix the docs of docker ps help

Signed-off-by: Xian Chaobo  xianchaobo@huawei.com

### CODE REVIEW

File: api/client/commands.go [hunk 1]
```diff
@@ -1611,7 +1611,7 @@ func (cli *DockerCli) CmdPs(args ...string) error {
 	)
 	cmd.Require(flag.Exact, 0)
 
-	cmd.Var(&flFilter, []string{"f", "-filter"}, "Filter output based on conditions provided")
+	cmd.Var(&flFilter, []string{"f", "-filter"}, "Filter output based on conditions provided.  Valid filters:\nid=<id> - containers with id of <id>\nname=<name> - containers with name of <name>\nexited=<int> - containers with exit code of <int>\nstatus=(restarting|running|paused|exited)")
```
Comment (duglin) [inline api/client/commands.go]: Help text for options can no go past 80 chars and must stay on just one line.
Additional details should then be in the docs.

**Comment by duglin** [comment]
please squash your commits

**Comment by duglin** [comment]
If you run the integration-cli test: TestHelpTextVerify  you'll see the test that catches this stuff

File: api/client/commands.go (hunk 1 above)
Comment (jimmyxian) [inline api/client/commands.go]: OK, i will just fix the docs of docker ps help. do i need to open a new PR?

Comment (duglin) [inline api/client/commands.go]: no, just update the current commits.  You may also want to check, I'm pretty sure there are other spots in the docs where the additional text describing the current filters would be good - like the man pages but @moxiegirl would know for sure.

Comment (jimmyxian) [inline api/client/commands.go]: the man pages, i have checked, it's OK

**Comment by moxiegirl** [comment]
@jimmyxian Your signature is failing.  Gordon (our bot) expects a format like this:

Signed-off-by: FIRSTNAME LASTNAME username@company.com

You've only provided a single name, need both your given and your family name. 

http://docs.docker.com/project/set-up-git/#set-your-signature-and-an-upstream-remote

**Comment by thaJeztah** [comment]
Actually I suspect the signature is failing because the sign-off is using parentheses (`(` and `)`) in stead of `<` and `>`, but setting up Git correctly should work.

**Comment by jimmyxian** [comment]
OK, Thks @moxiegirl  @thaJeztah ,I will modify my signature

**Comment by jimmyxian** [comment]
@moxiegirl , I modified my signature, but also failed. do i need fork, update the docs and make new PR one more time?

**Comment by thaJeztah** [comment]
@jimmyxian your PR now has 4 commits; the "sign-off" check is performed on each individual commit. If you "squash" your commits to a single commit and have a correct signature on that, it should work.

If you need help, you can ask for assistance in the #docker or #docker-dev IRC channel, there's probably someone there that's able to help or give additional hints

**Pull request closed without merging**